tests/python/unittest/test_sparse_operator.py

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 from mxnet.test_utils import *
-from common import setup_module, with_seed
+from common import setup_module, with_seed, teardown
```

Comment:
applying this change would allow all tests within a module to finish before moving onto the next test, thus eliminating the case where side effect of tests in another module spills over to the next. In terms of testing practice, including a setup/teardown is common.

Reply:
Yeah, but we're not actually using it in most files, right?

Reply:
now we are

Reply:
Ah in common.py :) But isn't it sufficient to import it there?

Reply:
unfortunately no. it is the same case as setup_module

Reply:
argh :/